README.md

```diff
@@ -88,6 +88,18 @@ Run `carthage update --cache-builds` then drag & drop from the Carthage/Builds f
 - `RxNimbleRxBlocking.framework` and `RxBlocking.framework`
 - `RxNimbleRxTest.framework` and `RxTest.framework`
 
+## Migration 4.5.0 -> 5.0.0
+
+Deprecated function `equalFirst` were completely removed in favor of more `Nimble`-like style of API. Please use:
```

Comment:
What do you think of something like this? 

`equalFirst` was deprecated in favor of a more natural Nimble matcher API style. 

RxNimble 4.5.0: 

```swift
...
```

RxNimble 5.0.0:
```swift
...
```

Reply:
Sure, fixed